examples/Hermitian_Phasing.py

```diff
@@ -0,0 +1,102 @@
+"""
+Phasing and Timing Correction
+=============================
+
+Take fake data with a relatively symmetric echo 
+(:math:`T_2^*=1/50\pi`, echo time of 10 ms),
+and demonstrate how we can automatically find the zeroth order phase and the
+center of the echo in order to get data that's purely real in the frequency
+domain.
+"""
+from pyspecdata import *
+from pyspecProcScripts import *
+from pylab import *
+import sympy as s
+from collections import OrderedDict
+from numpy.random import normal, seed
+
+seed(2021)
+rcParams["image.aspect"] = "auto"  # needed for sphinx gallery
+
+# sphinx_gallery_thumbnail_number = 5
+
+t2, td, vd, power, ph1, ph2 = s.symbols("t2 td vd power ph1 ph2")
+echo_time = 10e-3
+f_range = (-400, 400)
+
+with figlist_var() as fl:
+    for expression, orderedDict, signal_pathway, indirect, label in [
+        (
+            (
+                23
+                * (1 - 2 * s.exp(-vd / 0.2))
+                * s.exp(+1j * 2 * s.pi * 100 * t2 - abs(t2) * 50 * s.pi)
+            ),
+            [
+                ("vd", nddata(r_[0:1:40j], "vd")),
+                ("ph1", nddata(r_[0:4] / 4.0, "ph1")),
+                ("ph2", nddata(r_[0, 2] / 4.0, "ph2")),
+                ("t2", nddata(r_[0:0.2:256j] - echo_time, "t2")),
+            ],
+            {"ph1": 0, "ph2": 1},
+            "vd",
+            "IR",
+        ),
+        (
+            (
+                23
+                * (1 - (32 * power / (0.25 + power)) * 150e-6 * 659.33)
+                * s.exp(+1j * 2 * s.pi * 100 * t2 - abs(t2) * 50 * s.pi)
+            ),
+            [
+                ("power", nddata(r_[0:4:25j], "power")),
+                ("ph1", nddata(r_[0:4] / 4.0, "ph1")),
+                ("t2", nddata(r_[0:0.2:256j] - echo_time, "t2")),
+            ],
+            {"ph1": 1},
+            "power",
+            "enhancement",
+        ),
+    ]:
+        fl.basename = "(%s)" % label
+        fig, ax_list = subplots(1, 4, figsize=(20, 20))
```

Comment:
I got rid of the figsize because it was cutting off the x axis name and so was slightly confusing. Without the figsize it looks good on my end